test_conformance/api/test_kernels.cpp

```diff
@@ -704,3 +722,65 @@ REGISTER_TEST(negative_set_immutable_memory_to_writeable_kernel_arg)
 
     return TEST_PASS;
 }
+
+REGISTER_TEST(negative_set_read_write_image_arg)
+{
+    cl_int error = CL_SUCCESS;
+    clProgramWrapper program;
+    clKernelWrapper kernels[2];
+    clMemWrapper images[2], buffer;
+    const char *test_kernels[2] = { sample_read_only_image_test_kernel,
+                                    sample_write_only_image_test_kernel };
+    constexpr cl_image_format formats = { CL_RGBA, CL_UNSIGNED_INT8 };
```

Comment:
Nitpick: This is a single format, not multiple formats.

```suggestion
    constexpr cl_image_format format = { CL_RGBA, CL_UNSIGNED_INT8 };
```

Reply:
Done